src/main/java/adry/earthquake/EarthquakeFrame.java

```diff
@@ -0,0 +1,90 @@
+package adry.earthquake;
+
+import hu.akarnokd.rxjava3.swing.SwingSchedulers;
+import io.reactivex.rxjava3.disposables.Disposable;
+import io.reactivex.rxjava3.schedulers.Schedulers;
+import adry.earthquake.json.Feature;
+import adry.earthquake.json.FeatureCollection;
+
+import javax.swing.*;
+import java.awt.*;
+import java.awt.event.ActionListener;
+
+public class EarthquakeFrame extends JFrame {
+
+    private JList<String> jlist = new JList<>();
+
+    public EarthquakeFrame() {
+
+        setTitle("Earthquake Frame");
+        setSize(300, 600);
+        setDefaultCloseOperation(EXIT_ON_CLOSE);
+
+        setLayout(new BorderLayout());
+        JRadioButton hour;
+        JRadioButton month;
+
+        hour = new JRadioButton("One Hour");
+        hour.addActionListener(e -> {
+            adry.earthquake.EarthquakeService service = new EarthquakeServiceFactory().getService();
```

Comment:
Create this only once.

Reply:
make one action listener that can be applied to both buttons?

Reply:
No. Create only one EarthquakeService